trace.go

```diff
@@ -27,6 +27,8 @@ type TraceInfo struct {
 	// ConnTime is a duration that took to obtain a successful connection.
 	ConnTime time.Duration
 
+	TCPConnTime time.Duration
```

Comment:
This needs a godoc comment.

Reply:
Added!

Reply:
Needs an ending period.  😃 Best practice is to use complete sentences for godoc comments.  Also, capitalize TCP.

Reply:
Thank you for your comment @moorereason !

Added in this new commit!